client/a8c-for-agencies/sections/purchases/licenses/license-details/index.tsx

```diff
@@ -121,19 +113,6 @@ export default function LicenseDetails( {
 					</li>
 				) }
 			</ul>
-
```

Comment:
Can we delete the `LicenseDetailsActions` component since it is not being used now.